flashinfer/jit/attention.py

```diff
@@ -19,13 +19,16 @@
 from typing import List, Tuple
 
 import jinja2
+import rich.syntax
```

Comment:
What's the purpose of import this module?

Reply:
removed.